Neos.Cache/Classes/Backend/MultiBackend.php

```diff
@@ -221,10 +223,10 @@ public function collectGarbage()
     }
 
     /**
-     * @param array $backendConfigurations
+     * @param mixed[] $backendConfigurations
```

Comment:
Huh, that's an odd way to denote `array` 🤨

Reply:
Yup, wouldn't do that. I get `string[]` to make clear what is expected, but `mixed[]` is in no way better than `array` IMHO.

Plus: currently a type like `string[]` breaks Doctrine, so using this in entities would be bad. So let's keep that notation out for now, please. Avoids confusion.

Reply:
Then again… `string[]` really is helpful. (But `mixed[]`… not so much.)